application/Modules/Invoices/Models/InvoiceAmount.php

```
@@ -109,7 +113,7 @@ public function calculate($invoice_id, $global_discount)
         $this->calculate_invoice_taxes($invoice_id);
 
         // Get invoice status
-        $this->load->model('invoices/mdl_invoices');
+        $this->load->model('invoices/invoice');
         $invoice           = $this->mdl_invoices->get_by_id($invoice_id);
         $invoice_is_credit = ($invoice->creditinvoice_parent_id > 0);
```

PHPMD (2.15.0)

The variable $invoice_is_credit is not named in camelCase. (undefined)

(CamelCaseVariableName)